oscar_stripe_sca/templates/oscar_stripe_sca/stripe_waiting.html

```diff
@@ -0,0 +1,54 @@
+{% extends 'oscar/checkout/payment_details.html' %}
+{% load i18n static currency_filters %}
+
+{% block checkout_title %}
+    {% translate "Checkout" %}
+{% endblock %}
+
+{% block checkout_nav %}
+    {% include 'oscar/checkout/nav.html' with step=4 %}
+{% endblock %}
+
+{% block order_contents %}{% endblock %}
+{% block shipping_address %}{% endblock %}
+{% block shipping_method %}{% endblock %}
+{% block payment_method %}{% endblock %}
+
+{% block payment_details_content %}
+    <p>Waiting for order confirmation, please wait...</p>
+
+    <script type="text/javascript">
```

Comment:
I'd prefer the script in a separate static file. This allows for more secure CSP and easier overriding. The settings (URLs and polling interval) can be injected into the page as a JSON script: https://docs.djangoproject.com/en/4.2/ref/templates/builtins/#json-script.